contents/blog/soft-delete/index.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: Using the New Soft-delete Feature
+date: 2022-10-17
+author: Maciej Obuchowski & Michael Robinson
+image: ./image.svg
+banner: ./banner.svg
```

Comment:
On the first blog post the banner was 765x123. Same deal here - since there are only two posts, we have options on how to make them both match :)